src/main/resources/assets/botania/lang/zh_tw.lang

```diff
@@ -1,3742 +1,3815 @@
-item.botania:cacophonium.name=不和諧之音
-item.botania:slimeBottle.name=史萊姆罐子
-item.botania:starSword.name=召星者
-item.botania:exchangeRod.name=移殼桿 
-item.botania:magnetRingGreater.name=巨型磁化環
-item.botania:thunderSword.name=召雷者
-item.botania:manaweaveCloth.name=魔源布匹
-item.botania:manaweaveHelm.name=魔源頭蓬
-item.botania:manaweaveChest.name=魔源長袍
-item.botania:manaweaveLegs.name=魔源長袍(下半身)
-item.botania:manaweaveBoots.name=魔源長靴
-item.botania:santaweaveHelm.name=聖誕編織斗篷
-item.botania:santaweaveChest.name=聖誕編織長袍
-item.botania:santaweaveLegs.name=聖誕編織長袍(下半身)
-item.botania:santaweaveBoots.name=聖誕編織長靴
-item.botania:gaiaHead.name=蓋亞守護者的頭
-item.botania:sextant.name=蓋世六分儀
-item.botania:speedUpBelt.name=平原旅帶
-item.botania:baubleBox.name=飾品盒子
-item.botania:manaPowder.name=瑪那塵
-
-# BREW NAMES
-botania.brew.fallback=退卻
-botania.brew.speed=飛毛腿
-botania.brew.strength=活力
-botania.brew.haste=腎上腺素
-botania.brew.healing=治療
-botania.brew.jumpBoost=上湧
-botania.brew.regen=再生
-botania.brew.regenWeak=恢復
-botania.brew.resistance=剛毅
-botania.brew.fireResistance=炎膚
-botania.brew.waterBreathing=魚鰓
-botania.brew.invisibility=隱身
-botania.brew.nightVision=梟眼
-botania.brew.absorption=護盾
-botania.brew.soulCross=噬魂
-botania.brew.featherFeet=羽翼
-botania.brew.emptiness=虛空之影
-botania.brew.bloodthirst=腥紅之影
-botania.brew.overload=過載
-botania.brew.allure=誘惑
-botania.brew.warpWard=理智
-botania.brew.clear=赦免
-
-# POTION NAMES
-botania.potion.soulCross=噬魂
-botania.potion.featherFeet=羽翼
-botania.potion.emptiness=空虛
-botania.potion.bloodthirst=嗜血
-botania.potion.allure=引誘
-botania.potion.clear=赦免
-
-# GUI TEXT
-botania.gui.lexicon.header=植物魔法辭典 第%s版 - 目錄
-botania.gui.lexicon.wip=(本條目的編纂工作仍在進行,其當前內容仍未完全定稿)
-
-# MUSIC DISCS
-item.record.botania:gaia1.desc=Kain Vinosec - Endure Emptiness
-item.record.botania:gaia2.desc=Kain Vinosec - Fight For Quiescence
-
-# BANNER NAMES
-item.banner.botania_flower.black=黑色神秘花
-item.banner.botania_flower.red=紅色神秘花
-item.banner.botania_flower.green=綠色神秘花
-item.banner.botania_flower.brown=棕色神秘花
-item.banner.botania_flower.blue=藍色神秘花
-item.banner.botania_flower.purple=紫色神秘花
-item.banner.botania_flower.cyan=青色神秘花
-item.banner.botania_flower.silver=淺灰色神秘花
-item.banner.botania_flower.gray=灰色神秘花
-item.banner.botania_flower.pink=粉色神秘花
-item.banner.botania_flower.lime=淺綠色神秘花
-item.banner.botania_flower.yellow=黃色神秘花
-item.banner.botania_flower.lightBlue=淺藍色神秘花
-item.banner.botania_flower.magenta=洋紅色神秘花
-item.banner.botania_flower.orange=橘色神秘花
-item.banner.botania_flower.white=白色神秘花
-
-item.banner.botania_lexicon.black=黑色辭典
-item.banner.botania_lexicon.red=紅色辭典
-item.banner.botania_lexicon.green=綠色辭典
-item.banner.botania_lexicon.brown=棕色辭典
-item.banner.botania_lexicon.blue=藍色辭典
-item.banner.botania_lexicon.purple=紫色辭典
-item.banner.botania_lexicon.cyan=青色辭典
-item.banner.botania_lexicon.silver=淺灰色辭典
-item.banner.botania_lexicon.gray=灰色辭典
-item.banner.botania_lexicon.pink=粉色辭典
-item.banner.botania_lexicon.lime=淺綠色辭典
-item.banner.botania_lexicon.yellow=黃
```

Comment:
all of these Japanese names are Vocaloid song names. Are you sure that the Chinese names you have are the official names for those Vocaloid songs?